src/ActivityLogger.php

```diff
@@ -30,18 +30,21 @@ class ActivityLogger
     /** @var \Illuminate\Support\Collection */
     protected $properties;
 
+    /** @var string */
+    protected $authDriver;
```

Comment:
Of course. Failed to see in the GitHub code review that the changes were in two seperate functions. My bad.